GWDetChar test suite fails with pygments 2.19.0

The HTML tests for this project fail when running with the latest releases of pygments:

```shell
conda create -n test python=3.11 gwdetchar pygments=2.19 pytest
python3 -m pytest --pyargs gwdetchar.io.tests.test_html
```

This results in the following test failures:

```pytest
======================================== test session starts =========================================
platform linux -- Python 3.11.11, pytest-8.3.4, pluggy-1.5.0
rootdir: /home/duncan/git/git.ligo.org/computing/software/dqsegdb2
configfile: pyproject.toml
collected 27 items

. .......FF..................                                                                  [100%]

============================================== FAILURES ==============================================
________________________________________ test_about_this_page ________________________________________

package_list = <MagicMock name='package_list' id='139627750194704'>
tmpdir = local('/tmp/pytest-of-duncan/pytest-400/test_about_this_page0')

    @mock.patch(
        "gwdetchar.io.html.package_list",
        return_value=[
            {"name": "gwpy", "version": "1.0.0"},
            {"name": "gwdetchar", "version": "1.2.3"},
        ],
    )
    def test_about_this_page(package_list, tmpdir):
        outdir = str(tmpdir)
        config_file = os.path.join(outdir, 'test.ini')
        with open(config_file, 'w') as fobj:
            fobj.write(TEST_CONFIGURATION)
        testargs = ['/opt/bin/gwdetchar-scattering',
                    '-i', 'X1']
        with mock.patch.object(sys, 'argv', testargs):
            # test with a single config file
            about = html.about_this_page(config_file)
>           assert parse_html(about) == parse_html(ABOUT)
E           AssertionError: assert 'Start tag: d...nd tag: div\n' == 'Start tag: d...nd tag: div\n'
E
E             Skipping 463 identical leading characters in diff, use -v to show
E             -  'color: #bbbbbb')
E             +  'color: #BBB')
E               Data:
E               End tag: span
E               Data: gwdetchar-scattering...
E
E             ...Full output truncated (142 lines hidden), use '-vv' to show

/home/duncan/opt/conda/envs/test/lib/python3.11/site-packages/gwdetchar/io/tests/test_html.py:365: AssertionError
_______________________________________ test_get_command_line ________________________________________

    def test_get_command_line():
        testargs = ['/opt/bin/gwdetchar-conlog',
                    '--html-only',
                    '-i', 'X1']
        with mock.patch.object(sys, 'argv', testargs):
            cmdline = str(html.get_command_line())
>           assert (f'gwdetchar-conlog{pygments_space_span}-i'
                    f'{pygments_space_span}X1') in cmdline
E           assert 'gwdetchar-conlog<span style="color: #bbbbbb"> </span>-i<span style="color: #bbbbbb"> </span>X1' in '<p>This page was generated with the following command-line call:</p>\n<div class="highlight" style="background: #f8f8...lor: #BBB"> </span>X1\n</pre></div>\n\n<p>The install path used was <code>/home/duncan/opt/conda/envs/test</code>.</p>'

/home/duncan/opt/conda/envs/test/lib/python3.11/site-packages/gwdetchar/io/tests/test_html.py:379: AssertionError
====================================== short test summary info =======================================
FAILED ::test_about_this_page - AssertionError: assert 'Start tag: d...nd tag: div\n' == 'Start tag: d...nd tag: div\n'
FAILED ::test_get_command_line - assert 'gwdetchar-conlog<span style="color: #bbbbbb"> </span>-i<span style="color: #bbbbbb"> </sp...
==================================== 2 failed, 25 passed in 1.15s ====================================
```

The fundamental issue is a change to pygments to shorten hex colour codes, see https://github.com/pygments/pygments/commit/0853028bfe180b7c56ef20502f44a5d8d340216c.

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -61,10 +61,8 @@ jobs:
       uses: conda-incubator/setup-miniconda@v3
       with:
         auto-update-conda: true
-        channels: conda-forge
+        miniforge-version: latest
         python-version: ${{ matrix.python-version }}
-        # this is needed for caching to work properly:
-        use-only-tar-bz2: true
 
     - name: Conda info
       run: conda info --all

--- a/gwdetchar/io/tests/test_html.py
+++ b/gwdetchar/io/tests/test_html.py
@@ -27,7 +27,6 @@ import sys
 
 from getpass import getuser
 from MarkupPy import markup
-from pygments import __version__ as pygments_version
 from pytz import reference
 from unittest import mock
 
@@ -72,34 +71,15 @@ NEW_BOOTSTRAP_PAGE = """<!DOCTYPE HTML>
 TEST_CONFIGURATION = """[section]
 key = value"""
 
-if pygments_version >= "2.14.0":
-    pygments_space_span = '<span style="color: #bbbbbb"> </span>'
-    pygments_output = (
-        '\n'
-        '<span style="color: #687822">key</span>'
-        f'{pygments_space_span}'
-        '<span style="color: #666666">=</span>'
-        f'{pygments_space_span}'
-        '<span style="color: #BA2121">value</span>'
-    )
-elif pygments_version >= "2.11.0":
-    pygments_output = (
-        '<span style="color: #bbbbbb"></span>\n'
-        '<span style="color: #687822">key</span>'
-        '<span style="color: #bbbbbb"> </span>'
-        '<span style="color: #666666">=</span>'
-        '<span style="color: #bbbbbb"> </span>'
-        '<span style="color: #BA2121">value</span>'
-        '<span style="color: #bbbbbb"></span>'
-    )
-    pygments_space_span = ' '
-else:
-    pygments_output = (
-        '\n'
-        '<span style="color: #7D9029">key</span> '
-        '<span style="color: #666666">=</span> '
-        '<span style="color: #BA2121">value</span>'
-    )
+pygments_space_span = '<span style="color: #BBB"> </span>'
+pygments_output = (
+    '\n'
+    '<span style="color: #687822">key</span>'
+    f'{pygments_space_span}'
+    '<span style="color: #666">=</span>'
+    f'{pygments_space_span}'
+    '<span style="color: #BA2121">value</span>'
+)
 
 ABOUT = f"""<div class="row">
 <div class="col-md-12">

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -63,6 +63,7 @@ dynamic = ["version"]
 [project.optional-dependencies]
 test = [
   "flake8",
+  "pygments >=2.19.0",
   "pytest >=3.3.0",
   "pytest-cov >=2.4.0",
 ]

